providers/bnxt_re/db.c

```diff
@@ -40,6 +40,75 @@
 #include <stdlib.h>
 #include "main.h"
 
+static uint32_t xorshift32(struct xorshift32_state *state)
```

Comment:
This function is called only once.

Reply:
> This function is called only once.

isn't it okay to keept it as it is? It is implementing the xorshift algo and I feel its better to keep it as a separate function.  Is it sufficient to mark it as an inline function?

Reply:
no inline ,please.

maybe add that function to util/ folder.

Reply:
> no inline ,please.
> 
> maybe add that function to util/ folder.

Sure.. will do.